src/test/java/jewellery/inventory/unit/service/SaleServiceTest.java

```diff
@@ -180,20 +176,19 @@ void testReturnProductWillThrowsSaleNotFoundException() {
 
   @Test
   void testReturnProductSuccessfullyWithTwoProducts() {
```

Comment:
Why did we change this test? It no longer tests what happens with a sale when there are multiple (2) products inside it.